docs/test_e2e.md

```diff
@@ -2,39 +2,39 @@
 
 ## Table of Contents
 
-- [Installing a specific VMClarity build on AWS](#installing-a-specific-vmclarity-build-on-aws)
+- [Installing a specific OpenClarity build on AWS](#installing-a-specific-openclarity-build-on-aws)
   - [1. Build the containers and publish them to your Docker registry](#1.-build-the-containers-and-publish-them-to-your-docker-registry)
-  - [2. Install VMClarity CloudFormation](#2.-install-vmclarity-cloudformation)
-  - [3. Ensure that VMClarity backend is working correctly](#3.-ensure-that-vmclarity-backend-is-working-correctly)
+  - [2. Install OpenClarity CloudFormation](#2.-install-openclarity-cloudformation)
+  - [3. Ensure that OpenClarity backend is working correctly](#3.-ensure-that-openclarity-backend-is-working-correctly)
 - [Performing an end-to-end test](#performing-an-end-to-end-test)
 
-## Installing a specific VMClarity build on AWS
+## Installing a specific OpenClarity build on AWS
 
 ### 1. Build the containers and publish them to your Docker registry
 
 ```shell
 DOCKER_REGISTRY=<your docker registry> make push-docker
 ```
 
-### 2. Install VMClarity CloudFormation
+### 2. Install OpenClarity CloudFormation
 
 1. Ensure you have an SSH key pair uploaded to AWS EC2
 2. Go to CloudFormation -> Create Stack -> Upload template.
-3. Upload the VMClarity.cfn
+3. Upload the OpenClarity.cfn
 4. Follow the wizard through to the end
-    - Set the `VMClarity Backend Container Image` and `VMClarity Scanner Container Image` parameters in the wizard to
-    use custom images (from step 1.) for deployment.
-    - Change the Asset Scan Delete Policy to `OnSuccess` or `Never` if debugging scanner VMs is required.
+   - Set the `OpenClarity Backend Container Image` and `OpenClarity Scanner Container Image` parameters in the wizard to
+     use custom images (from step 1.) for deployment.
+   - Change the Asset Scan Delete Policy to `OnSuccess` or `Never` if debugging scanner VMs is required.
 5. Wait for install to complete
 
-### 3. Ensure that VMClarity backend is working correctly
+### 3. Ensure that OpenClarity backend is working correctly
 
 1. Get the IP address from the CloudFormation stack's Output Tab
 2. `ssh ubuntu@<ip address>`
-3. Check the VMClarity Logs
+3. Check the OpenClarity Logs
 
    ```shell
-   sudo journalctl -u vmclarity
+   sudo journalctl -u openclarity
```

Comment:
The service name is still `vmclarity` here, this is one of the missing renames along with logs and filepaths https://github.com/openclarity/openclarity/pull/756#issue-2463390832. But I do not believe this is a blocker, I can update this after this PR is merged

Reply:
Thanks for noticing!